### PR TITLE
request for favicon addition

### PR DESCRIPTION
**Pull request by 2u841r**

### CODE REVIEW

**Review by 2u841r — COMMENTED** [review]
I just noticed there was a svg logo

**Comment by 2u841r** [comment]
the svg is also black!

File: index.html [hunk 1]
```diff
@@ -7,6 +7,8 @@
 	<link rel="preconnect" href="https://fonts.gstatic.com"/>
 	<link rel="preload" href="https://fonts.googleapis.com/css2?family=Roboto&family=Roboto+Slab:wght@300&display=swap" as="style"/>
 	<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&family=Roboto+Slab:wght@300&display=swap" media="print" onload="this.media='all'"/>
+	<!-- 	Please consider adding a favicon to make it look nicer -  https://i.ibb.co/wY41kQP/favicon.png -->
+    	<link rel="icon" type="image/svg+xml" href="logo.svg">
```
Comment (fregante) [inline index.html:11]: The can you fix the indentation and remove the comment? Code comments are not the right place for requests

**Review by fregante — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (2u841r) [reply index.html]: done

File: logo2.svg
Comment (fregante) [inline logo2.svg:1]: I’m not really a fan of how it looks. Let’s just use the existing black version for now

Comment (2u841r) [reply logo2.svg:1]: Ok. Please change and merge.

Comment (2u841r) [reply logo2.svg:1]: logowhitebg
how about this? with white background, instead of transparent background?

Comment (fregante) [reply logo2.svg:1]: It's not square and the problem is that adding padding makes it even smaller in the tab. That's why I prefer just setting a favicon for now and then I'll figure it out later

**Pull request closed without merging**